src/content/LandingPage/_landing-page.scss

```diff
@@ -0,0 +1,45 @@
+@import './mixins.scss';
+@import './overrides.scss';
+
+.landing-page__illo {
+  max-width: 100%;
+}
+
+.landing-page__banner {
+  padding-top: $spacing-05;
+  padding-bottom: $spacing-07 * 4;
```

Comment:
I think you may have missed the `@include landing-page-background` in this step of the tutorial.  The first row/banner area should have a gray background similar to the third row.

Reply:
Fixed, thanks for the review.